src/modal/modal-ref.ts

```diff
@@ -16,7 +16,7 @@ export class NgbActiveModal {
   close(result?: any): void {}
 
   /**
-   * Can be used to dismiss a modal, passing an optional reason.
+   * Can be used to dismiss a modal, passing an optional reason (result is discarded).
```

Comment:
I'm sorry, but it is not clear to me what you mean here... What is the `result` that you are mentioning? What do you mean by it being discarded? 

As it stands I find it more confusing that the original description...

Could you please provide more context / info?

Reply:
@pkozlowski-opensource took the liberty to refine docs further. It wasn't clear for me either :)